### ISSUE TITLE
Styling the vertical scroll bar in panel

### ISSUE THREAD

**Issue by quadrismegistus** [Princeton-CDH/lenape-timetree#160]
It looks a little awkward to me as is:

![Screen Shot 2023-04-19 at 10 19 30 AM](https://user-images.githubusercontent.com/733853/233104874-611696e5-409a-4e9c-8b77-fe740b5d97c5.png)

**Comment by rlskoeser** [comment]
@quadrismegistus which browser?

**Comment by quadrismegistus** [comment]
Chrome

**Comment by quadrismegistus** [comment]
Here is Firefox:

![Screen Shot 2023-04-19 at 10 23 38 AM](https://user-images.githubusercontent.com/733853/233105703-b94c5ea2-373a-4d35-9505-cd38a86acf67.png)

**Comment by rlskoeser** [comment]
Yes, I've been noticing that the padding over the scrollbar is kind of annoying. The x is positioned correctly, I was trying to give as much padding as possible so it would be a bigger touch/click target. But we should probably shift some of that padding to margin to avoid overlapping with the scrollbar.

I forget, how much control do we have over styling the scrollbar? Is it still inconsistent / limited / browser dependent?

**Comment by quadrismegistus** [comment]
Looks like it works everywhere but safari on iOS? https://caniuse.com/css-scrollbar

Requires two sets of CSS rules though: https://www.digitalocean.com/community/tutorials/css-scrollbars
